src/components/HeartbeatBar.vue

```diff
@@ -229,4 +303,17 @@ export default {
     }
 }
 
+.word {
+    color: #aaa;
+    font-size: 12px;
+}
+
+.connecting-line {
+    flex-grow: 1;  // To make the line expand and fill the available space
```

Comment:
This comment is just stating what `flex-grow: 1` odes => I don't think it holds additional value ^^
```suggestion
    flex-grow: 1;
```